tools/mdb_to_wt.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import bson, codecs, pprint, subprocess, sys
+
+# Decodes a MongoDB file into a readable format.
+def util_usage():
+    print("Usage: mdb_to_wt <path_to_wt> filename")
```

Comment:
The name of the script may not be correct? I got it as it converts something from MDB to WT. How about `wt_to_mdb_bson`?

Reply:
@kommiharibabu My understanding is that none of our WT tests generate anything in BSON format, hence this script is only used for MDB files e.g. indexes and collections. Is that correct?

Reply:
Yes, that's correct, none of the WT tests doesn't generate BSON format data and also I agree that this script is used to decode the data stored by MongoDB in WT. 

As this script is converting the data stored in WT to MDB understanding BSON format, I suggested that name.

Reply:
Ah yes, I see what you mean. I will change the name to the name you suggested.